.github/workflows/extension-ci.yml

```
@@ -0,0 +1,143 @@
+        env:
+          FORCE_COLOR: 1
+
+      - name: Upload Extension Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: extension-package
+          path: |
+            apps/extension/vsix-build/*.vsix
+            apps/extension/dist/
+          retention-days: 30
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)